### ISSUE TITLE
Command CompileSwift failed with a nonzero exit code

### ISSUE THREAD

**Issue by monish152** [stripe/stripe-ios#1806]
I have added stripe new sdk in objective c project. Its working fine on simulator.

While archiving II am getting below error.

Command CompileSwift failed with a nonzero exit code
Framework not found Stripe

I have onesignal extension. Xcode is latest.

Please let me know if anyone facing issue.

**Comment by aliriaz-stripe** [comment]
@monish152 hello!

Can you share more info, like: 

1/ how you integrated stripe-ios? Cocoapods? a dynamic framework? 

2/ the version of stripe-ios you are on

2/ a screenshot of the error, if possible

3/ your version of XCode

**Comment by monish152** [comment]
Hi @aliriaz-stripe 

1. I added by cocoapods
2. I think its latest..in my pods its have pod 'Stripe'. I have updated my cocoapods also
3. Sure attached is error code
4. xcode version 12.0
5. My pod file attached
<img width="634" alt="Screenshot 2021-05-13 at 8 31 59 AM" src="https://user-images.githubusercontent.com/1301491/118071606-bade6b80-b3c5-11eb-937e-460ee75ac7e8.png">

6. 
<img width="830" alt="Screenshot 2021-05-13 at 8 29 41 AM" src="https://user-images.githubusercontent.com/1301491/118071559-a4d0ab00-b3c5-11eb-8ac6-a315264af947.png">
